lusSTR/repeat.py

```diff
@@ -49,18 +49,22 @@ def collapse_all_repeats(sequence, repeats):
     return collapsed_seq
 
 
-def split_by_n(sequence, n):
+def split_by_n(sequence, n, rev):
```

Comment:
Would it make sense to set the default value for rev=True?

Reply:
Good implementation of split_by_n. I am surprised a function like this doesn't already exist in python but I couldn't find it. This is something I could see being reused in other projects.

Reply:
I guess the default value should by False not True.

Reply:
I added default value of False.